src/pollux/utils/JWT.ts

```diff
@@ -18,7 +20,39 @@ export class JWT {
     this.castor = castor;
   }
 
-  private async resolve(did: string): Promise<didResolver.DIDResolutionResult> {
+  async verify(
+    options: {
+      issuerDID: DID,
+      holderDID?: DID,
+      jws: string
+    }
+  ): Promise<boolean> {
+    try {
+      const { issuerDID, jws, holderDID } = options;
+      const resolved = await this.resolve(issuerDID.toString());
+      const verificationMethod = resolved.didDocument?.verificationMethod;
+      if (!verificationMethod) {
+        throw new Error("Invalid did document");
```

Comment:
Replace with a custom error.

Reply:
Same, this is inside a try catch and the error is ignored